invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/index.js

```diff
@@ -57,9 +58,9 @@ class DashboardTabs extends Component {
       _camelCase("communities-enabled")
     );
 
-    let ACTIVE_TAB_PANES = communitiesEnabled ? TAB_PANES : [TAB_PANES[0]];
+    this.ACTIVE_TAB_PANES = communitiesEnabled ? TAB_PANES : [TAB_PANES[0]];
```

Comment:
Thanks for that! I had fixed it locally but it seems I never pushed it to the PR that was merged :(